deploy/docker-compose.yml

```diff
@@ -47,36 +58,15 @@ services:
         command:
             -   'echo "plone image $$IMAGE_ID built"'
 
-    volto-vawag:
```

Comment:
Shall we keep this to facilitate the multiple analytics accounts?

Reply:
Sure, though I'd like to move these to GKE and we can do that in there instead.